.changeset/giant-eyes-tie.md

```diff
@@ -0,0 +1,7 @@
+---
+"eppo_core": major
```

Comment:
Thanks for making this a major bump - makes a ton of sense to give us room to test python and ruby more extensively.

Reply:
It doesn't give us much room. It is going to bump eppo_core's version in both python and ruby, so the next python/ruby releases will release with this new change. (This is the same regardless of whether it's a major or patch change.)